scrypto/src/component/stubs.rs

```diff
@@ -382,10 +387,13 @@ extern_blueprint_internal! {
         fn quick_confirm_recovery_role_badge_withdraw_attempt(&mut self) -> Bucket;
         fn cancel_primary_role_badge_withdraw_attempt(&mut self);
         fn cancel_recovery_role_badge_withdraw_attempt(&mut self);
-        fn mint_recovery_badges(&mut self, non_fungible_local_ids: Vec<NonFungibleLocalId>) -> Bucket;
+        fn mint_recovery_badges(
```

Comment:
This should return a `NonFungibleBucket` and not a fungible bucket since the recovery badge is non-fungible.

Reply:
Done